src/lib/coaler/multialign/MultiAligner.cpp

```diff
@@ -262,6 +272,9 @@ namespace coaler::multialign {
             omp_unset_lock(&bestAssemblyScoreLock);
         }
         spdlog::info("finished alignment optimization.");
+        if (skippedAssembliesCount > 0) {
+            spdlog::info("Skipped a total of {} incomplete assemblies.", skippedAssembliesCount);
+        }
```

Comment:
Finde ich nice. So gibt es die information noch in info, aber verbraucht nicht mehr so viel Platz